src/io/flutter/view/FlutterView.java

```diff
@@ -199,12 +207,23 @@ private void addInspectorPanel(String displayName,
    * Called when a debug connection starts.
    */
   public void debugActive(@NotNull FlutterViewMessages.FlutterDebugEvent event) {
+    final FlutterApp app = event.app;
+
+    whenCompleteUiThread(InspectorService.create(app.getFlutterDebugProcess(), app.getVmService()),
+                         (InspectorService inspectorService, Throwable throwable) -> {
+                           if (throwable != null) {
+                             LOG.error(throwable);
```

Comment:
I think we want `warn` here, so we don't show a dialog to the user.

Reply:
done